server/src/main/scala/actors/AlertsActor.scala

```diff
@@ -19,6 +20,8 @@ case class AlertsActor(now: () => SDateLike) extends RecoveryActorLike with Pers
   override val snapshotBytesThreshold: Int = oneMegaByte
   override val maybeSnapshotInterval: Option[Int] = None
 
+  override val recoveryStartMillis: MillisSinceEpoch = now().millisSinceEpoch
+
   override def processRecoveryMessage: PartialFunction[Any, Unit] = {
     case alert: ProtobufAlert =>
       log.info(s"ALERTS GOT alert $alert")
```

Comment:
looks like we should probably delete this log line while you are here.

Reply:
good call!